### ISSUE TITLE
An issue for false-positive FP detection

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/FingerprintingBlocker#5]
Let's keep this issue open, I'll be adding websites in comments.

**Comment by ameshkov** [comment]
github.com

**Comment by ameshkov** [comment]
roem.ru

**Comment by theseanl** [comment]
These can actually be a correct detection based on what a user considers a "Canvas fingerprinting" is.
Most of such false-positive is caused by websites trying to detect different levels of emoji support by printing some unicode text in the canvas and reading pixels.

While this is a 'browser feature detection', this can also be regarded as a 'browser fingerprinting' using canvas, and a reality is that most of other extensions that promises to block canvas fingerprinting is detecting such practice as a fingerprinting.

To distinguish such practice with more outright fingerprinting (e.g. as done in github.com/valve/fingerprintjs2), I'm considering showing notifications only when a number of pixels read are greater than certain number, or when a number of modified pixels (for faking mode) are greater than certain number.

**Comment by ameshkov** [comment]
> I'm considering showing notifications only when a number of pixels read are greater than certain number, or when a number of modified pixels (for faking mode) are greater than certain number.

Makes sense. I'd still like to have this issue open with all the websites. We can use that list to test your solution later.

**Comment by ameshkov** [comment]
trello.com -- breaks favicon

**Comment by ameshkov** [comment]
betanews.com -- _wpemojiSettings

**Comment by theseanl** [comment]
kt.com - cause is a library [conic-gradient](https://github.com/leaverou/conic-gradient).
